docs/api/rest/index.rst

```diff
@@ -40,7 +40,7 @@ If all you want is reference guides, skip straight to :ref:`rest-api-schemas`.
 
 .. versionchanged:: 3.1
 
-   The API version was bumped to v1.3 in Patchwork v3.1. The older APIs are
+   The API version was bumped to v1.4 in Patchwork v3.1. The older APIs are
    still supported. For more information, refer to :ref:`rest-api-versions`.
```

Comment:
```suggestion
.. versionchanged:: 3.1

   The API version was bumped to v1.3 in Patchwork v3.1. The older APIs are
   still supported. For more information, refer to :ref:`rest-api-versions`.

.. versionchanged:: 3.2

   The API version was bumped to v1.4 in Patchwork v3.2. The older APIs are
   still supported. For more information, refer to :ref:`rest-api-versions`.
```

Reply:
Fixed, I wasn't sure if I should bump the application version alongside the API version :sweat_smile: